pkg/openstack/rabbitmq.go

```diff
@@ -65,6 +87,13 @@ func ReconcileRabbitMQ(ctx context.Context, instance *corev1beta1.OpenStackContr
 			rabbitmq.Spec.Override.StatefulSet = &defaultStatefulSet
 		}
 
+		if rabbitmq.Spec.Rabbitmq.AdditionalConfig == "" {
+			helper.GetLogger().Info("Setting AdditionalConfig")
+			// This is the same situation as RABBITMQ_UPGRADE_LOG above,
+			// except for the "main" rabbitmq log we can just force it to use the console.
+			rabbitmq.Spec.Rabbitmq.AdditionalConfig = "log.console = true"
```

Comment:
only question here is should we append this config option if for example the user supplies their own AdditionalConfig via the rabbitmqTemplate?

Reply:
Yeah good point, I was torn about that.  It's mildly annoying that AdditionalConfig is just a plain string instead of something more structured, which makes it harder to parse and reason about.  I was worried about the case where the user supplied something like:

```
"foo = bar
log.console = false
log.syslog = true
# maybe even throw in a comment for good measure
baz = baz"
```

At that point we probably don't want to just blindly append `log.console = true`, we'd need to parse it and fully understand the contents and only set `log.console = true` if the user hasn't already supplied some other value.

In the end I went with the easier approach for now, with the (probably poor) assumption that if the user is overriding the default but they still want to keep the default, then it's up to them to include it in whatever config they supply.